dev-util/radare2/radare2-5.4.0.recipe

```diff
@@ -277,7 +277,7 @@ INSTALL()
 	mv $prefix/data/radare2 $docDir
 	rm -rf $prefix/share
 
-	prepareInstalledDevelLibs libr_shlr libr_anal libr_asm libr_bin \
+	prepareInstalledDevelLibs libr_anal libr_asm libr_bin libr_debug libr_main \
 	libr_bp libr_config libr_cons libr_core libr_crypto libr_debug libr_egg \
```

Comment:
libr_debug is called twice here:

```diff
-	prepareInstalledDevelLibs libr_anal libr_asm libr_bin libr_debug libr_main \
+	prepareInstalledDevelLibs libr_anal libr_asm libr_bin \
```

`libr_main` should be moved further in the line (alphabetically)

Reply:
Should be fixed in https://github.com/haikuports/haikuports/pull/6185